app/js/form.js

```diff
@@ -0,0 +1,183 @@
+
+$('input,textarea').on('input',function(){
+  var inpObj1 = document.getElementById("name").checkValidity();
```

Comment:
make a class and then attach  event or functions with it. Directly attaching events to tag will have unexpected behavior in your app.